src/main/java/org/folio/linked/data/configuration/json/deserialization/event/InventoryInstanceEventDeserializer.java

```diff
@@ -0,0 +1,78 @@
+package org.folio.linked.data.configuration.json.deserialization.event;
+
+import static java.util.Objects.isNull;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.linked.data.domain.dto.InventoryInstance;
+import org.folio.linked.data.domain.dto.InventoryInstanceEvent;
+import org.folio.linked.data.domain.dto.ResourceIndexEventType;
+
+@Log4j2
+@RequiredArgsConstructor
+public class InventoryInstanceEventDeserializer extends JsonDeserializer<InventoryInstanceEvent> {
```

Comment:
is it really necessary to create a deserializer for InventoryInstanceEvent? I mean, ther should be specific reason, like for SRS event which contains json string instead of normal json thus it could not be automatically deserialized

Reply:
you're right, InventoryInstanceEventDeserializer is not required, event deserialization works put of the box, deleted this.